velox/functions/sparksql/tests/RegexFunctionsTest.cpp

```diff
@@ -301,15 +306,21 @@ TEST_F(RegexFunctionsTest, regexpReplaceWithEmptyString) {
 }
 
 TEST_F(RegexFunctionsTest, regexBadJavaPattern) {
-  EXPECT_THROW(testRegexpReplace("[]", "[a[b]]", ""), VeloxUserError);
-  EXPECT_THROW(testRegexpReplace("[]", "[a&&[b]]", ""), VeloxUserError);
-  EXPECT_THROW(testRegexpReplace("[]", "[a&&[^b]]", ""), VeloxUserError);
+  VELOX_ASSERT_THROW(
+      testRegexpReplace("[]", "[a[b]]", ""),
+      "regexp_replace does not support character class union, intersection, or difference ([a[b]], [a&&[b]], [a&&[^b]])");
```

Comment:
This test doesn't match documentation which says that the function returns NULL in this case.